oobi/benchmark-definitions/python/data_types.py

```diff
@@ -66,6 +74,19 @@ class ModelArtifact(object):
   source_url: str
 
 
+@serialization.serializable
+@dataclass(frozen=True)
+class MetaModel(object):
```

Comment:
How about use `Model` for this and rename the existing `Model` to `ExecutionBenchmark`?

Reply:
Oh, I misunderstood the code. This is a higher-level abstraction for models with different batch sizes.